geodepy/constants.py

```diff
@@ -233,322 +233,609 @@ class parameters.
+    tx=-0.08468, ty=-0.01942, tz=0.03201,
+    sc=0.00971,
+    rx=-0.0004254, ry=0.0022578, rz=0.0024015,
+    d_tx=0.00142, d_ty=0.00134, d_tz=0.00090,
+    d_sc=0.000109,
+    d_rx=0.0015461, d_ry=0.0011820, d_rz=0.0011551,
+    tf_sd=itrf2008_to_gda94_sd)
+
+itrf2005_to_gda94 = Transformation(
+    from_datum='ITRF2005', to_datum='GDA94', ref_epoch=date(1994, 1, 1),
+    tx=-0.07973, ty=-0.00686, tz=0.03803,
+    sc=0.006636,
+    rx=-0.0000351, ry=0.0021211, rz=0.0021411,
+    d_tx=0.00225, d_ty=-0.00062, d_tz=-0.00056,
+    d_sc=0.000294,
+    d_rx=0.0014707, d_ry=0.0011443, d_rz=0.0011701,
+    tf_sd=itrf2005_to_gda94_sd)
+
+itrf2000_to_gda94 = Transformation(
+    from_datum='ITRF2000', to_datum='GDA94', ref_epoch=date(1994, 1, 1),
+    tx=-0.04591, ty=-0.02985, tz=-0.02037,
+    sc=0.00707,
+    rx=-0.0016705, ry=0.0004594, rz=0.0019356,
+    d_tx=-0.00466, d_ty=0.00355, d_tz=0.01124,
+    d_sc=0.000249,
+    d_rx=0.0017454, d_ry=0.0014868, d_rz=0.001224,
+    tf_sd=itrf2000_to_gda94_sd)
+
+itrf97_to_gda94 = Transformation(
+    from_datum='ITRF97', to_datum='GDA94', ref_epoch=date(1994, 1, 1),
+    tx=-0.01463, ty=-0.02762, tz=-0.02532,
+    sc=0.006695,
+    rx=-0.0017893, ry=-0.0006047, rz=0.0009962,
+    d_tx=-0.00860, d_ty=0.00036, d_tz=0.01125,
+    d_sc=0.000007,
+    d_rx=0.0016394, d_ry=0.0015198, d_rz=0.0013801,
+    tf_sd=itrf97_to_gda94_sd)
+
+itrf96_to_gda94 = Transformation(
+    from_datum='ITRF96', to_datum='GDA94', ref_epoch=date(1994, 1, 1),
+    tx=0.02454, ty=-0.03643, tz=-0.06812,
+    sc=0.006901,
+    rx=-0.0027359, ry=-0.0020431, rz=0.0003731,
+    d_tx=-0.02180, d_ty=0.00471, d_tz=0.02627,
+    d_sc=0.000388,
+    d_rx=0.0020203, d_ry=0.0021735, d_rz=0.0016290,
+    tf_sd=itrf96_to_gda94_sd)
+
+agd84_to_gda94 = Transformation(
+    from_datum='AGD84', to_datum='GDA94', ref_epoch=0,
+    tx=-117.763, ty=-51.510, tz=139.061,
+    sc=-0.191,
+    rx=-0.292, ry=-0.443, rz=-0.277)
+
+agd66_to_gda94 = Transformation(
+    from_datum='AGD66', to_datum='GDA94', ref_epoch=0,
+    tx=-117.808, ty=-51.536, tz=137.784,
+    sc=-0.290,
+    rx=-0.303, ry=-0.446, rz=-0.234)
+
+agd66_to_gda94_act = Transformation(
+    from_datum='AGD66', to_datum='GDA94', ref_epoch=0,
+    tx=-129.193, ty=-41.212, tz=130.730,
+    sc=-2.955,
+    rx=-0.246, ry=-0.374, rz=-0.329)
+
+agd66_to_gda94_tas = Transformation(
+    from_datum='AGD66', to_datum='GDA94', ref_epoch=0,
+    tx=-120.271, ty=-64.543, tz=161.632,
+    sc=2.499,
+    rx=-0.217, ry=0.067, rz=0.129)
+
+agd66_to_gda94_vicnsw = Transformation(
+    from_datum='AGD66', to_datum='GDA94', ref_epoch=0,
+    tx=-119.353, ty=-48.301, tz=139.484,
+    sc=-0.613,
+    rx=-0.415, ry=-0.260, rz=-0.437)
+
+agd66_to_gda94_nt = Transformation(
+    from_datum='AGD66', to_datum='GDA94', ref_epoch=0,
+    tx=-124.133, ty=-42.003, tz=137.400,
+    sc=-1.854,
+    rx=0.008, ry=-0.557, rz=-0.178)
+
+gda19_to_itrf2008 = -itrf2008_to_gda94
```

Comment:
Should be gda94_to_itrf2008? Same typo made below as well

Reply:
@harry093 well spotted! All fixed up in latest commit